bakerydemo/static/css/main.css

```diff
@@ -385,6 +385,24 @@ li.has-submenu a.allow-toggle {
 .caret-custom:after {
     content: "▼"!important;
 }
+/* Mobile menu styling */
+@media (max-width: 768px) {
+  .nav-pills>li {
+    display: block;
+    float: none;
+    width: 100%!important;
+  }
+  .nav-pills> li > a {
+    border-top: 1px solid rgba(255,255,255,.1);
+    padding: 20px 0;
+  }
+  li.has-submenu a.allow-toggle {
+    float: none;
+  }
+  li.has-submenu a.caret-custom {
+    display: none!important;
+  }
```

Comment:
I don't see an example of those `.has-submenu` rules happening in the demo, could you point me to how to get this?

Reply:
They're not in the demo fixtures but if you were to add some of the sub-pages to the menu (via the 'Show in menus' in 'promote') and they'll be visible. This caret was an incredibly hacky way of presenting the dropdown on desktop. (The menu's constructed by base/templatetags/navigation_tags.py)

Reply:
Gotcha 👍. Just reviewed that change, works as expected.